TESTS/netsocket/nidd/niddsocket_echotest.cpp

```diff
@@ -129,6 +129,8 @@ void NIDDSOCKET_ECHOTEST_NONBLOCK()
         double loss_ratio = 1 - ((double)packets_recv / (double)packets_sent);
         tr_info("Packets sent: %d, packets received %d, loss ratio %.2lf", packets_sent, packets_recv, loss_ratio);
         TEST_ASSERT_DOUBLE_WITHIN(TOLERATED_LOSS_RATIO, EXPECTED_LOSS_RATIO, loss_ratio);
+    } else {
```

Comment:
I guess the `if (packets_sent > 0)` was here to avoid division by 0, not to detect that packets were sent or not. If they were not sent we would fail in line 117. So I think this is dead code.
Unless I am not seeing some failure mode, that in fact leads to this branch being a possible option?